i18n/en.json

```diff
@@ -219,7 +219,7 @@
   "admin.permissions.permission.edit_post.description": "Anytime after posting, allow users to edit their own posts.",
   "admin.permissions.permission.delete_post.name": "Delete Own Posts",
   "admin.permissions.permission.delete_post.description": "Authors own posts can be deleted.",
-  "admin.permissions.permission.delete_others_posts.name": "Delete Any Post",
+  "admin.permissions.permission.delete_others_posts.name": "Delete Others Posts",
   "admin.permissions.permission.delete_others_posts.description": "Any post can be deleted.",
```

Comment:
This should be "Posts made by other uses can be deleted."

Reply:
Do we need to make any changes to the behavior of the checkboxes, or are they already working as mutually exclusive options?